cli.js

```diff
@@ -297,6 +298,13 @@ const commands = [
         alias: ["pkv"],
         options: "-verbose|v",
         action: plonkVerify
+    },
+    {
+        cmd: "file info [binary.file] [zkey|ptau|r1cs|wtns]",
```

Comment:
```suggestion
        cmd: "file info [binary.file] [zkey|ptau|r1cs|wtns]",
```
This cmd sting is confusing. I believe it's indicating that `.file` should be `zkey` or `ptau` or `r1cs` or `wtns` but the general CLI syntax is `[optional arg 1] [optional arg 2]`

Can we reword?

Reply:
My mistake, in fact command string is cmd: "file info [binary.file]"